src/main/java/core/basesyntax/Engine.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
-public class Engine {
+public class Engine implements Cloneable {
 
+    private int levelOfNoise;
+    private int numOfCylinders;
+    private int power;
+    private String typeOfFuel;
+    private String typeOfCooling;
+
+    @Override
+    public Engine clone() throws CloneNotSupportedException {
+        return (Engine) super.clone();
```

Comment:
super clone will return the same object with a different link. Try to return new Engine